hr_timesheet_holiday/models/hr_holidays.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from openerp.exceptions import Warning as UserError
+
+
+class HrHolidaysStatus(models.Model):
+    """Add analytic account to holiday status"""
+    _inherit = 'hr.holidays.status'
+
+    analytic_account_id = fields.Many2one('account.analytic.account',
+        'Analytic Account')
```

Comment:
which seems correct to me. but only if there exist already timesheet lines written with this type